tasks/install-from-source.yml

```diff
@@ -36,11 +36,13 @@
   command: >
     {{ item }}
     chdir={{ workspace }}/ruby-{{ ruby_version }}
-    creates=/usr/local/bin/ruby
+    creates=/usr/local/bin/.ruby-{{ ruby_version }}
```

Comment:
This seems like it might pollute the bin dir — typically I don't like the idea of stashing user / data files inside any of my bin dirs.

Reply:
We can have it in the workspace / etc or var. Or the .ansible folder.  Wdyt ?

Reply:
Agree it's a bit weird to put it in a `bin` directory - maybe `/usr/local/lib/ansible/ruby-{{ruby_version}}` ? Or `/var/cache` ?  The latter might be purged sometimes, which I guess might lead to ansible rebuilding ruby unnecessarily.

Reply:
@jdelStrother What about the $HOME/.ansible/checks/ ?

Reply:
Yeah, I think I like that better than my options. You'd think ansible might have a convention for this sort of stuff, but I don't see it anywhere.

Reply:
I took a run at this here - https://github.com/jdelStrother/ansible-role-ruby/commit/8d54c0df6f30007ff0e2c2ed692a4121771e5f74 - can submit a PR if you like

Reply:
@jdelStrother  Thank for the ping. I forgot to push the change.
I end-up using /var/cache instead of the home folder.
Your commit has some additional improvement